<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>340</width>
    <height>697</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="Menu">
       <widget class="QPushButton" name="log_out">
        <property name="geometry">
         <rect>
          <x>220</x>
          <y>10</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(18, 140, 126);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
       <widget class="QListWidget" name="conectedUsers">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>291</width>
          <height>601</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(7, 94, 84);</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>201</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Chats</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Chat">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>550</y>
          <width>301</width>
          <height>71</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0" colspan="2">
          <widget class="QLineEdit" name="lnMessage">
           <property name="styleSheet">
            <string notr="true">color: rgb(7, 94, 84);</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="btnSend">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(18, 140, 126);
color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string>Send</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QListWidget" name="lstMessages">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>301</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="backmenu">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">alternate-background-color: rgb(18, 140, 126);
color: rgb(255, 255, 255);
background-color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Back</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>301</width>
          <height>35</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(7, 94, 84);</string>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="Login">
       <widget class="QLineEdit" name="lineEdit">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>240</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>User</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>170</y>
          <width>101</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Login</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLineEdit" name="lineEdit_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>290</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QPushButton" name="login">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>340</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(18, 140, 126);
color: rgb(255, 255, 255);</string>
        </property>
        <property name="text">
         <string>Log in</string>
        </property>
       </widget>
       <widget class="QPushButton" name="singin">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>370</y>
          <width>91</width>
          <height>29</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Sign in</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="Sign">
       <widget class="QLineEdit" name="Name">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>220</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="signin">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>390</y>
          <width>101</width>
          <height>29</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">alternate-background-color: rgb(7, 94, 84);

color: rgb(255, 255, 255);
background-color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Sign In</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>260</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>User </string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>150</y>
          <width>91</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>15</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Sign In</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>220</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QDateEdit" name="userdate">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>340</y>
          <width>111</width>
          <height>29</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>300</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>340</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Date</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Username">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>260</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="loginb">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>430</y>
          <width>101</width>
          <height>29</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(18, 140, 126);</string>
        </property>
        <property name="text">
         <string>Log In</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLineEdit" name="password">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>300</y>
          <width>113</width>
          <height>28</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page">
       <widget class="QLineEdit" name="lnClientName">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>190</y>
          <width>428</width>
          <height>28</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>Enter Client Name</string>
        </property>
       </widget>
       <widget class="QComboBox" name="cmbDestination">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>300</y>
          <width>237</width>
          <height>28</height>
         </rect>
        </property>
        <property name="editable">
         <bool>true</bool>
        </property>
        <property name="currentText">
         <string>Server</string>
        </property>
        <item>
         <property name="text">
          <string>Server</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
       </widget>
       <widget class="QComboBox" name="cmbStatus">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>80</y>
          <width>261</width>
          <height>28</height>
         </rect>
        </property>
        <item>
         <property name="text">
          <string>None</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Available</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Away</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Busy</string>
         </property>
        </item>
       </widget>
       <widget class="QPushButton" name="btnSendFile">
        <property name="geometry">
         <rect>
          <x>130</x>
          <y>390</y>
          <width>35</width>
          <height>29</height>
         </rect>
        </property>
        <property name="maximumSize">
         <size>
          <width>35</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>...</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>320</x>
          <y>510</y>
          <width>83</width>
          <height>29</height>
         </rect>
        </property>
        <property name="text">
         <string>chat</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>340</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="separator"/>
    <addaction name="actionExot"/>
   </widget>
   <addaction name="menuOptions"/>
  </widget>
  <action name="actionConnect">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionExot">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
